pulsar-zookeeper/src/main/java/org/apache/pulsar/zookeeper/ZooKeeperServerAspect.java

```diff
@@ -97,4 +92,27 @@ public double get() {
                     }
                 }).register();
     }
+
+    private Gauge.Child getDataSizeGauge(ZooKeeperServer zkServer) {
+        // off by default, for performance reasons
+        // to enable: pass -Dstats_use_actual_size=true to the JVM
+        boolean useActualSize = Boolean.parseBoolean(
+                System.getProperties().getProperty("stats_use_actual_size", "false"));
```

Comment:
knit: could simplify this to
```suggestion
        boolean useActualSize = Boolean.getBoolean("stats_use_actual_size");
```